npc/re/jobs/3-2/sura.txt

```diff
@@ -654,14 +654,14 @@ OnTimer305000:
 OnTimer315000:
 	mapannounce "sword_2-1","Come on here.",bc_map;
 	donpcevent "Buddy#job_shuaneh::OnDisable";
-	donpcevent "#Sura_garajjom::OnEnable";
+	// donpcevent "#Sura_garajjom::OnEnable";
 	end;
 OnTimer320000:
 	mapannounce "sword_2-1","I guess you're not ready to meet the master yet...",bc_map;
 	mapwarp "sword_2-1","yuno_fild07",255,178;
-	donpcevent "#Sura_garajjom::OnDisable";
-	donpcevent "Drawing Room::OnEnable";
+	// donpcevent "#Sura_garajjom::OnDisable";
```

Comment:
if those lines would be commented maybe should add a comment on them that explain why, otherwise if someone in the future compared the script with official one its may be confusing.